OpenEdXMobile/res/xml/remote_config_defaults.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<defaultsMap>
+    <entry>
+        <key>aa_experiment</key>
+        <value>Control</value>
```

Comment:
When/how does this default get used?

Can the value change based on network connectivity and experiment bucketing decisions?

Reply:
> When/how does this default get used?
If app unable to fetch remote configs( network issue ), then app use the default configurations. Once remote configs fetched app stores it for future use.

Reply:
For the purposes of our initial spike, can we make this some other value that isn't used in the actual experiment?

Maybe "Not Set"?

Reply:
> For the purposes of our initial spike, can we make this some other value that isn't used in the actual experiment?
> 
> Maybe "Not Set"?

Yes, we can add other values that are not part/set of/in the experiment.